about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About capstone</title>
+  <link rel="stylesheet" href="about.css">
+</head>
+<body>
+  <nav class="nav-box">
+    <div class="hamburger" onclick="toggleMenu()">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+
+  <ul class="dropdown-menu">
+    <li><a href="index.html">Home</a></li>
+    <li><a href="index.html#services">Services</a></li>
+    <li><a href="index.html#resident-musicians">Resident Musicians</a></li>
+    <li><a href="#about">About</a></li>
+  </ul>
+    <div class="logo">
+        <img src="Images/Logo.png" alt="">
+      </div>
+    <div class="dropdown-desktp">
+      <a href="index.html">Home</a>
+      <a href="index.html#services">Services</a>
+      <a href="index.html#resident-musicians">Resident-Musicians</a>
+       <a href="#about">About</a>
+    </div>
+ </nav>
+
+<section>
+<div class="about-head">
+  <h3>Hello, Creatives Welcome To</h3>
+  <h2>GROOVY STUDIOS.</h2>
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly try to replicate the design given as much as possible. Kindly ensure the hamburger icon is not aligned horizontally with these two headings.
- Your design
![your-mobile-nav](https://user-images.githubusercontent.com/63917725/233806011-f720371d-db2c-4368-9a1a-e409310d6f89.PNG)
- Given design
![given-mobile-nav](https://user-images.githubusercontent.com/63917725/233806025-4d1e41c1-9874-4c0f-a156-cc43070bcaa5.PNG)